igvc_ws/src/sensors/src/imu_node.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+
+'''
+    Description: Reads/Subsrcibes imu data, process the data to determine current velocity and delta distance
+    Date Modified: 1 Feb 2019
+    Author: Ben Brenkman
+'''
+
+
+import ros_api as ros
+from ros_api import println
+import rospy
+
+import custom_msgs.msg as msgs
+
+imu_data = None # last known imu data in JSON format
+
+velocity_x = 0.0
```

Comment:
I'm not sure about all these global variables (same with other scripts), they are generally a pretty bad idea, but it may be ok. We'll talk about it.